action_tests/all_test_payload.json

```diff
@@ -0,0 +1,19 @@
+{
+  "trunkPath": "./local-action/action_tests/stub.js",
+  "targetRefName": "main",
+  "targetCheckoutRef": "main",
+  "checkMode": "all",
+  "trunkToken": "fake-token",
+  "version": "0.0.0",
+  "arguments": "\" --replace\" : \"\"",
```

Comment:
ah, I think the bug is here - I think this should be just:
```
"arguments": "--replace"
```

Reply:
ohh i see ok